pkgs/by-name/xp/xppen/package.nix

```
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, fetchzip
+, autoPatchelfHook
+
+, libusb1
+, qt5
+, libGL
+}:
+stdenv.mkDerivation {
+  pname = "xppen";
```

See https://github.com/NixOS/nixpkgs/pull/285660#issuecomment-2411961311 for explanation

```suggestion
  pname = "xppen_3";
```